src/overlays/actors/ovl_Boss_03/z_boss_03.c

```diff
@@ -41,157 +266,2345 @@ const ActorInit Boss_03_InitVars = {
+                this->csState = 5;
+                this->csTimer = 0;
+                this->unk_2D5 = false;
+                this->actor.speedXZ = -200.0f;
+                Actor_MoveWithoutGravityReverse(&this->actor);
+                this->actor.world.pos.y = this->waterHeight - 150.0f;
+                func_80165690();
+
+                case 5:
+                    SkelAnime_Update(&this->skelAnime);
+                    this->actor.speedXZ = 20.0f;
+                    Actor_MoveWithoutGravityReverse(&this->actor);
+                    player->actor.shape.rot.y = -0x1470;
+                    player->actor.world.rot.y = player->actor.shape.rot.y;
+
+                    this->csCamEye.x = player->actor.world.pos.x + 30.0f - 90.0f + 300.0f;
+                    this->csCamEye.y = player->actor.world.pos.y + 40.0f + 10.0f;
+                    this->csCamEye.z = player->actor.world.pos.z - 30.0f + 160.0f + 300.0f;
+
+                    this->csCamAt.x = this->actor.world.pos.x;
+                    this->csCamAt.y = this->actor.world.pos.y - 100.0f;
+                    this->csCamAt.z = this->actor.world.pos.z;
+
+                    if (this->csTimer == 10) {
+                        this->actor.velocity.y = 30.0f;
+                        this->csState = 6;
+                        this->csTimer = 0;
+                        this->actor.gravity = -1.5f;
+                        this->actor.speedXZ = 20.0f;
+
+                        // Play the boss background music
+                        Audio_QueueSeqCmd(0x801B);
+                        Actor_PlaySfxAtPos(&this->actor, NA_SE_EN_KONB_JUMP_OLD);
+                        this->skelAnime.playSpeed = 1.0f;
+                    }
+            }
+            break;
+
+        case 6:
+            Actor_PlaySfxAtPos(&this->actor, NA_SE_EN_KONB_JUMP_LEV_OLD - SFX_FLAG);
+
+            if (this->csTimer == 30) {
+                TitleCard_InitBossName(&globalCtx->state, &globalCtx->actorCtx.titleCtxt,
+                                       Lib_SegmentedToVirtual(gGyorgTitleCardTex), 160, 180, 128, 40);
+            }
+
+            if ((this->csTimer < 24) || (this->csTimer >= 90)) {
+                SkelAnime_Update(&this->skelAnime);
+                Math_ApproachS(&this->actor.world.rot.x, this->actor.velocity.y * -300.0f, 3, 0x1000);
+                Actor_MoveWithGravity(&this->actor);
+                if ((this->actor.velocity.y <= 0.0f) && (this->actor.world.pos.y < (this->waterHeight + 50.0f))) {
+                    this->bubbleEffectSpawnNum = 2;
+                    this->actor.gravity = 0.0f;
+                    Math_ApproachZeroF(&this->actor.velocity.y, 1.0f, 1.0f);
+                    Math_ApproachZeroF(&this->actor.speedXZ, 1.0f, 0.5f);
+                } else {
+                    if (1) {}
+                    if (1) {}
+                    if (1) {}
+                    if (1) {}
+                    if (1) {}
+                    if (1) {}
```

Comment:
Really?

Reply:
Yes, really. I hate it

Reply:
I tested it again, and I was able to remove 3 from the 6 `if (1)`. I don't know what changed, because last time all of them were required